src/Views/Comparator.elm

```diff
@@ -121,17 +122,29 @@ addToComparison session label query =
                         }
                     )
 
+        Bookmark.Object objectQuery ->
+            objectQuery
+                |> ObjectSimulator.compute session.db
+                |> Result.map
+                    (\_ ->
+                        { complementsImpact = Impact.noComplementsImpacts
+                        , impacts = Impact.empty
+                        , label = label
+                        , stepsImpacts = Impact.noStepsImpacts
+                        }
```

Comment:
Why don't we assign the computed impacts to `impacts` here?

Reply:
Good question 😅 Fixed in the latest commit.